centos-7-and-extras-x86_64.cfg

```diff
@@ -1,7 +1,7 @@
 config_opts['root'] = 'centos-and-extras-7-x86_64'
 config_opts['target_arch'] = 'x86_64'
 config_opts['legal_host_arches'] = ('x86_64',)
-config_opts['chroot_setup_cmd'] = 'install bash bzip2 coreutils cpio diffutils system-release findutils gawk gcc gcc-c++ grep gzip info make patch redhat-rpm-config rpm-build sed shadow-utils tar yum-utils unzip util-linux which xz python'
+config_opts['chroot_setup_cmd'] = 'install bash bzip2 coreutils cpio diffutils system-release findutils gawk gcc gcc-c++ grep gzip info make patch redhat-rpm-config rpm-build sed shadow-utils tar yum-utils unzip util-linux which xz python goversioninfo'
```

Comment:
do we still need this if we don't build origin rpm?

Reply:
We do not. Will remove.